qemu/tests/uefi_pkg.py

```diff
@@ -63,9 +79,17 @@ def check_element(filename, file_list):
     error_context.context("Check the 'filename' elements in both json"
                           " files point to valid files.", test.log.info)
     for meta_file in meta_files:
+        test.log.info("Checking the meta file '%s'" % meta_file)
         with open(meta_file, "r") as f:
             content = json.load(f)
         filename = content["mapping"]["executable"]["filename"]
-        check_element(filename, output)
+        check_element_filename(filename, output)
+        # for amdsev and inteltdx json files, check the 'mode' element
+        # and because they don't have 'nvram-template' element,
+        # skip the 'nvram-template' element checking
+        if "mode" in content["mapping"]:
```

Comment:
It looks mode element is only used for amdsev and inteltdx, is that true?
I mean is there any possibility that a json file contains both mode and nvrom-template? If so, we missed the nvram-template check

Reply:
If a json file contains mode, it means that the nvram template is not included. Now it's only used for amdsev and inteltdx. Thanks.